store/tikv/gcworker/gc_worker.go

```diff
@@ -258,11 +258,19 @@ func (w *GCWorker) leaderTick(ctx context.Context) error {
 		return nil
 	}
 
+	concurrency, err := w.loadGCConcurrencyWithDefault()
```

Comment:
Better get how many stores the cluster has from PD, and use the number of stores as the concurrency.